ccloud-monitoring/docs/consumer-scenarios/consumer-lag-problem.rst

```diff
@@ -0,0 +1,164 @@
+.. _ccloud-monitoring-consumer-connectivity-problem:
+
+Consumer Lag Problem
+********************
+
+Consumer group lag is a tremendous performance indicator. It tells you the difference between the
+producer's last produced message and the consumer group's last commit. If you are unfamiliar with
+consumer groups or concepts like committing offsets, please refer to this
+`Kafka Consumer documentation <https://docs.confluent.io/platform/current/clients/consumer.html>`__.
+
+A large or quickly growing lag indicates that the consumer is not able to keep up with
+the volume of messages on a topic.
+
+This scenario will look at Confluent Cloud metrics from the Metrics API, `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ metrics, and
```

Comment:
Please use `|ccloud|` instead of `Confluent Cloud`, here and wherever else is applicable

Reply:
Idk why that is such a hard one for me to remember 😓  will correct throughout these additions

Reply:
corrected in latest commit f3f5b8f